src/main/resources/application.yaml

```diff
@@ -58,6 +58,31 @@ stargate:
           url: https://us-central1-aiplatform.googleapis.com
           api-key: YOUR_API_KEY
 
+        google:
```

Comment:
Why is it a different section, should be under `vertexai`

Reply:
Because I am not sure about the final structure and just added another one temporarily. Which URL should I use? If I change to `https://us-central1-aiplatform.googleapis.com/v1/projects/{PROJECT_ID}/locations/us-central1/publishers/google/models/{MODEL}:predict`, will it affect the service now? Should I remove the `api-key`?

Reply:
Also, should I check every other model’s docs to add more config details now so I can remove `ServiceConfig`?